app/src/main/java/com/example/simplecalculator/Calculator.kt

```diff
@@ -80,52 +80,7 @@ fun Calculator(
                     }
                 )
             }
-            Row(
```

Comment:
We can apply dry principles here. This can be extracted as a function that takes an array list with labels and renders row